FSHingTrip.js

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env node
 
-import path from "path";
-import fs from "fs-extra";
-import diff from "diff";
-import { execSync } from "child_process";
+const path = require("path")
+const fs = require("fs-extra")
+const diff = require("diff")
+const execSync = require("child_process").execSync;
```

Comment:
I think we could have `;` here right? After all the `require` lines.

Reply:
Well, it's official. I've become too reliant on prettier.